Client/ViewModels/SettingsPageViewModel.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Messenger_Client.Views;
+using Microsoft.Toolkit.Mvvm.Input;
+using System;
+using System.Windows.Input;
+using Windows.Storage;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Messenger_Client.ViewModels
+{
+    class SettingsPageViewModel
+    {
+        ApplicationDataContainer localSettings = ApplicationData.Current.LocalSettings;
```

Comment:
This could probably be in the app.xaml.cs or client.cs and that the applications access the settings from there?

Reply:
Not much use I don't think... It's just that it's shorter to type, but doesn't offer much functionality. Instead, we've chosen to set the default settings in App.xaml.cs and leave the field in the settings viewmodel for convenience.